CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## [2.3.3] = TODO
```

Comment:
I'll update this when this is approved and I know around when we'll release this hotfix.